topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,30 @@
+package strings
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of runes.
+	runes := []rune(str)
+
+	// Here we create int that will be a pointer to the front of the runes.
+	var beg int
+
+	// Here  we create int that will be a pointer to the end of the runes.
```

Comment:
// Create an index that will traverse the collection of runes from the end to the beginning.